MobileCenter/MobileCenter/Internals/Channel/MSLogManagerDefault.m

```diff
@@ -91,7 +91,7 @@ - (void)processLog:(id<MSLog>)log withPriority:(MSPriority)priority {
   id<MSChannel> channel = [self channelForPriority:priority];
 
   // Set common log info.
-  log.toffset = [NSNumber numberWithInteger:[[NSDate date] timeIntervalSince1970]];
+  log.toffset = [NSNumber numberWithInteger:([[NSDate date] timeIntervalSince1970] * 1000)];
```

Comment:
We should add a macro or helper for the whole ([[NSDate date] timeIntervalSince1970] * 1000) thing. Otherwise we'll fall into it at some other points ;)